src/test/java/teammates/test/cases/logic/StudentsLogicTest.java

```diff
@@ -587,17 +587,16 @@ public void testEnrollLinesChecking() throws Exception {
     }
 
     /**
-     * This method returns the Exception message thrown when trying to build StudentList from invalid enrollment line.
-     * The method assumes that an Enrollment Exception is thrown, else the method fails with
-     * signalFailureToDetectException().
+     * Returns the Exception message thrown when trying to build StudentList from invalid enrollment line.
```

Comment:
Without using 'return' here the sentence is complete and hard to understand. This method is in a way a get method, so inevitably I have to use either return or get in my method description.